### DISCUSSION TITLE
Updating System.Resources.Extensions to 8.0.0 in Roslyn repo

### DISCUSSION THREAD

**Discussion by tmat** [dotnet/source-build#4202] [General]
I'm trying to update System.Resources.Extensions to 8.0.0 in Roslyn repo.
 
It seems like SystemResourcesExtensionsVersion not overridden by source build, so the build fails due to additional pre-built:
 
[Update VS SDK to 17.10.22-preview-1 by tmat · Pull Request #72359 · dotnet/roslyn (github.com)](https://github.com/dotnet/roslyn/pull/72359/checks?check_run_id=22188946659)

**Comment by mthalman** [comment]
I've responded: https://github.com/dotnet/roslyn/pull/72359#issuecomment-1989201620

In the future, you can ping the source build team directly from the PR using @dotnet/source-build-internal
